source/Reviewers/howtocommit.rst

```diff
@@ -464,15 +464,19 @@ for all affected tests before you commit to the trunk.
         latter case, the update will need redoing by the reviewer before commit
         if there are merge conflicts in the checksum files.
 
-        1. Run the rose stem tasks that require a KGO update, plus any other testing required (see above) - if unsure run the `all` group.
+        1. Fix any merge conflicts in the checksums - select the "Working" panel of the merge to keep the kgo files that are currently on trunk.
```

Comment:
I'm not sure about including the instruction to select "working" since people use different conflict resolution tools, and you're about to overwrite the file so it doesn't matter which you select. Maybe swap it for something saying that it doesn't matter?

Reply:
Yep fair point, have modified